nav2_bringup/bringup/README.md

```diff
@@ -62,26 +62,31 @@ To view the robot model in RViz:
 
 ## Launch Navigation2 on a Robot
 
-Pre-requisites:
-* Run SLAM or Cartographer with tele-op to drive the robot and generate a map of an area for testing first. The directions below assume this has already been done. If not, it can be done in ROS1 before beginning to install our code.
+### Pre-requisites:
+* Run SLAM with Navigation 2 or tele-op to drive the robot and generate a map of an area for testing first. The directions below assume this has already been done or there is already a map of the area.
+
+    Learn more about how to use Navigation 2 with SLAM to create maps; 
+    - [Navigation 2 with SLAM](https://github.com/ros-planning/navigation2/blob/master/doc/use_cases/navigation_with_slam.md)
+
+
 * Publish all the transforms from your robot from base_link to base_scan
 
-Launch the code using this launch file and your map.yaml:
+### Terminal 1 : Launch Navigation 2 using your map.yaml
 
 `ros2 launch nav2_bringup nav2_bringup_launch.py map:=<full/path/to/map.yaml> map_type:=occupancy`
 
-In another terminal, run RVIZ:
+### Terminal 2 : Launch RVIZ
 
 `ros2 run rviz2 rviz2 -d $(ros2 pkg prefix nav2_bringup)/share/nav2_bringup/launch/nav2_default_view.rviz`
 
 In RVIZ:
 * Make sure all transforms from odom are present. (odom->base_link->base_scan)
 * Localize the robot using “2D Pose Estimate” button.
-
-In RVIZ:
-* Localize the robot using “2D Pose Estimate” button.
 * Send the robot a goal using “2D Nav Goal” button.
 
-## Future Work
+## WARNINGS - Please Read
 
-* Add instructions for running navigation2 with SLAM
\ No newline at end of file
+Notes: (September 2019, Dashing Release)
+* We recommend doing this on a Ubuntu 18.04 installation. We have build issues on 16.04. (see https://github.com/ros-planning/navigation2/issues/353)
+* This stack and ROS2 are still in heavy development and there are some bugs and stability issues being worked on, so please do not try this on a robot without taking *heavy* safety precautions. THE ROBOT MAY CRASH!
```

Comment:
I think actually all of these notes can go away. 

You should never run code on a robot you don't understand, that's just a general rule of not putting holes in your expensive walls.

Reply:
I agree with you. I don't think we should have these warnings in this tutorial. We can move them to some other doc file in the repo or completely remove them.

Reply:
I'd just remove them.